project2-beat-that/script.js

```diff
@@ -1,4 +1,65 @@
+//for main function
+var player_score1 = 0;
+var player_score2 = 0;
+var playerFinalScore = 0;
+
+//players current score
+var player1_score = 0;
+var player2_score = 0;
+
+// players total score
+var player1_total = 0;
+var player2_total = 0;
+
+// counters
+var counter = 0;
+var remainder = 0;
+var player2 = 0;
+
+// arrray for score recording
+var allPlayersTotal = [0, 0];
```

Comment:
nice and clear global variables. take note that javascript naming convention is camelCase. you're using it for some variables but then using snake_case for others